### DISCUSSION TITLE
Need maintainers with knowledge about running spark on Azure, Databricks, Amazon Cloud and Google Cloud.

### DISCUSSION THREAD

**Discussion by quanghgx** [nightscape/spark-excel#521] [General]
As we can see from the issues queue, there are a number of reported issues when running spark-excel on cloud platforms. Those we do not have readily available access nor expertise. For example: https://github.com/crealytics/spark-excel/labels/cloud

Need help/maintainers, with access/knowledge about running spark on cloud platforms, eg: Azure, Databricks, Amazon Cloud and Google Cloud.

Sincerely,

**Comment by cedtnee** [comment]
@quanghgx  available for Google Cloud.

**Reply by quanghgx** [reply]
Awesome @cedtnee !
Hi @nightscape Please help suggest needed steps/permissions for @cedtnee

**Reply by nightscape** [reply]
Sorry for taking so long to respond, this one slipped off my radar.
@cedtnee I've given you `Triage` rights, which allows you to e.g. tag or close issues.
A helpful step would be to tag issues regarding Google Cloud with a corresponding tag.
If you feel like it, it would be nice to do the same thing for other cloud providers as well.
Another helpful addition would be to copy the [newly created issue template](https://github.com/crealytics/spark-excel/blob/main/.github/ISSUE_TEMPLATE/generic.yml) and adapt it to query specific details for Google Cloud.
